packages/react-searchbox/src/components/SearchBox.js

```diff
@@ -98,7 +99,7 @@ class SearchBox extends React.Component {
   get popularSuggestionsList() {
     const suggestions = this.componentInstance.suggestions;
     return (suggestions || []).filter(
-      suggestion => suggestion._popular_suggestion
+      suggestion => suggestion.source._popular_suggestion
```

Comment:
@anik-ghosh-au7 Why did you change it?

Reply:
@bietkul 

https://www.loom.com/share/55ec82016b6f47e48e4bfae8f6cda9c5